impl/src/test/java/org/ehcache/impl/persistence/FileBasedStateRepositoryTest.java

```diff
@@ -104,7 +104,7 @@ public void testIndexProperlySetAfterLoad() throws Exception {
 
     assertThat(files.length, is(1));
   }
-  
+
```

Comment:
Ooops.

Reply:
Well, this is required to clean that otherwise checkstyle is failing.